.gitignore

```diff
@@ -45,6 +45,7 @@ Thumbs.db
 .next
 .vscode
 .env
+.env.local
```

Comment:
Remove this extra inclusion.

Reply:
i added with comparison to the sentry doc's. Anyways adding that would not disturb any configs

Reply:
Cool, we can let it stay.